micr_XPCTzarr/participants.json

```
@@ -0,0 +1,36 @@
+{
+    "participant_id": {
+        "Description": "Unique alphanumeric participant ID starting with sub-"
+    },
+    "participant_name": {
+        "Description": "full name or pseudo of the participant which can contain non-alphanumeric characters"
+    },
+    "sex": {
+        "Description": "sex of the participant",
+        "Levels": {
+            "M": "male",
+            "F": "female"
+        }
+    },
+    "age": {
+        "Description": "age of the participant",
+        "Units": "year"
+    },
+    "weight": {
+        "Description": "weight of the participant",
+        "Units": "kilogram"
+    },
+    "height": {
+        "Description": "height of the participant",
+        "Units": "centimeter"
```

Following [CMIXF-12](https://people.csail.mit.edu/jaffer/MIXF/CMIXF-12):

```suggestion
        "Units": "kg"
    },
    "height": {
        "Description": "height of the participant",
        "Units": "cm"
```